src/components/dnn-permissions-grid/localisation-interface.tsx

```diff
@@ -0,0 +1,11 @@
+export interface ILocalization{
+    Add: string;

```

Comment:
```suggestion
    add: string;
```

Reply:
I can do this change, but if I do we need to change the DNN APIs, because this is what DNN returns (the localization key from the actual resx file)... We can chat later when you are back. But I think it is better to leave this way as I believe the built-in localization mechanism in the persona bar will need to be adjusted or the consumer will need different interfaces to make Uppercase to camelCase each and every property and in javascript objects, casing is just a code-style and we don't have listing rules to enforce cameCase.

Reply:
Awesnap - well...that was a complete waste of time.  😂👍🏼